public/userInfo.php

```diff
@@ -144,10 +143,36 @@
         'userMassData' => $userMassData,
         'userClaims' => $userClaimData?->toArray(),
     ]);
-    echo "</div>";
+
+    $canShowProgressionStatusComponent =
+        !empty($userCompletedGamesList)
+        // Needs at least one non-event game.
+        && count(array_filter($userCompletedGamesList, fn ($game) => $game['ConsoleID'] != 101)) > 0;
+
+    if ($canShowProgressionStatusComponent) {
+        echo "<div class='mt-1 mb-8 bg-embed p-5 rounded'>";
+        echo Blade::render('
+            <x-user-progression-status
+                :userCompletionProgress="$userCompletionProgress"
+                :userJoinedGamesAndAwards="$userJoinedGamesAndAwards"
+                :userSiteAwards="$userSiteAwards"
+                :userRecentlyPlayed="$userRecentlyPlayed"
+                :userHardcorePoints="$userHardcorePoints"
+                :userSoftcorePoints="$userSoftcorePoints"
+            />
+        ', [
+            'userCompletionProgress' => $userCompletedGamesList,
+            'userJoinedGamesAndAwards' => $userJoinedGamesAndAwards,
+            'userSiteAwards' => $userAwards,
+            'userRecentlyPlayed' => $userMassData['RecentlyPlayed'],
+            'userHardcorePoints' => $userMassData['TotalPoints'],
+            'userSoftcorePoints' => $userMassData['TotalSoftcorePoints'],
+        ]);
+        echo "</div>";
+    }
 
     if (isset($user) && $permissions >= Permissions::Moderator) {
-        echo "<div class='devbox mt-8'>";
+        echo "<div class='devbox mt-0'>";
```

Comment:
Maybe we could move the button up into the header and have the section expand between the header and the Most Recently Played:

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/edcd4262-f02e-4dcc-960e-f00248690ea7)

Reply:
> But I don't see why. Progression Status - "Most Recent" in particular - is tied to the Last X Games Played.

My perception is the original placement now disrupts the flow of information on the page:

![Screenshot 2024-02-03 at 6 43 23 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/2070398f-ba80-4aba-bfcd-4d4d66a65f9b)

> Maybe we could move the button up into the header and have the section expand between the header and the Most Recently Played

I like this and think it's a good idea. However, I'm finding this to be nontrivial -- solely because all of the other content has been converted to Blade with the exception of this admin box.

I'd like to punt this to a follow-up PR which migrates the admin box to Blade and repositions it to your suggested placement.

Reply:
I know this only affects a few users, but it does disrupt the flow no matter where you put it. I more closely associate the progression status widget with the recent games, which are now separated by it.

There really isn't a good place to inject it in the main portion of the profile page, which is what let me to suggesting putting it in the top block. A separate PR would be fine.

Reply:
👍 I will begin work on a separate PR for this tomorrow morning.